src/components/BorrowerProfile/SupportedByLenders.vue

```diff
@@ -76,15 +80,3 @@ export default {
 	},
 };
 </script>
-
-<style scoped lang="postcss">
-.challenge-avatar {
-	@apply tw-w-4;
-}
-
-.challenge-avatar :deep(div,)
-.challenge-avatar :deep(img),
-.challenge-avatar :deep(.loading-placeholder) {
```

Comment:
Wondering if we need to keep these for the placeholders which have hard-coded tw h + w classes in the component... Guessing the placeholder would be smaller in this case...